src/components/ContentNode/Grid.vue

```diff
@@ -0,0 +1,55 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2022 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+<template>
+  <div class="grid" :style="style" :class="{ 'with-columns': columns }">
+    <slot />
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'grid',
```

Comment:
```suggestion
  name: 'Grid',
```

Have you considered naming this `Row` to have it more closely map to the actual DocC markdown directive that it corresponds to? (Alternatively, maybe the directive should be called Grid to match this...)

Reply:
Yeah honestly I dont know which one is better 🤔 It feels like the Vue component should be `Grid`, but then do we call the `Column`s, `Cell`s?

Or do I just call it `Row` and call it a day?

Reply:
You may want to talk to Ethan about possibly renaming the directive to `Grid` if that's what makes the most sense—I think the directive name and component name would make sense to match each other unless there's a good reason otherwise.

Reply:
It's called a `Row` in Swift-DocC because it defines an individual row. You can't stack multiple cells vertically in it. If the Vue component can contain multiple Swift-DocC rows then I can see the argument for it being called a grid.

```md
This is my grid:

@Row {
  @Column {
    Bla.
  }
  @Column {
    Bla.
  }
}
@Row {
  @Column {
    Bla.
  }
  @Column {
    Bla.
  }
}
@Row {
  @Column {
    Bla.
  }
  @Column {
    Bla.
  }
}
```

Reply:
> but then do we call the Columns, Cells?

Technically the `@Column` directive is more defining a cell- I think for SwiftDocC it's much more intuitive to think about a column belonging to a row just from an authoring perspective. But maybe for the Vue component a `cell` makes more sense.

Reply:
Sounds like maybe we should rename it to `Row` since this is intended to be used only in support of that directive.

Reply:
Renamed